keyboards/handwired/ziyoulang_k3_mod/info.json

```
@@ -0,0 +1,140 @@
+{
+    "keyboard_name": "ziyoulang k3 mod",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
```

`readme.md` mentions development board used
```suggestion
    "development_board": "bluepill",
```

---

It was necessary to define a development board. I'll take this in.